.github/workflows/ur-build-hw.yml

```diff
@@ -96,7 +96,9 @@ jobs:
 
     - name: Install UR python dependencies
       working-directory: ${{github.workspace}}/unified-runtime
-      run: pip install -r third_party/requirements.txt
+      run: |
+        pip install pip==25.0.1
```

Comment:
For some reason, the CUDA build node was playing up until I added this. It's probably worth keeping it to ensure we always use the same version of pip.

Reply:
@lukaszstolarczuk perhaps we can update the host?

Reply:
Done. I think we should be good. Please remove and we'll see if it works now.